app/javascript/packs/stylesheets/bootstrap.scss

```diff
@@ -195,10 +118,64 @@ select.form-control {
   font-weight: 250 !important;
 }
 
+.nav-tabs .nav-link {
+  border-top-left-radius: 0 !important;
+  border-top-right-radius: 0 !important;
+}
+
+.nav-tabs .nav-link:hover {
+  border-color:#b5b5b5;
+  background-color:white;
+}
+
 .nav-link-inactive {
   color: rgba(255, 255, 255, 0.8) !important;
 }
 
+.nav-item:last-child {
+  margin-left: auto;
+}
+
+/* styling for navbar on smaller screens only */
+@media only screen and (max-width : 1310px) {
```

Comment:
It'd be nice if this breakpoint was more dynamic, so we don't have to worry about modifying it if we change any of the tabs. At the same time, if that would be overly complex to implement, it's probably not worth it.

I'll defer to you on the complexity, but if it is too complex, I would just ask that you split out the styling so we can have a separate `max-width` for each dashboard. The current value is appropriate for exposure, but it's unnecessarily wide for isolation. This is the breakpoint:

![image](https://user-images.githubusercontent.com/8235974/121904489-3bf78e00-ccf7-11eb-809f-3b37ec909160.png)

This discrepancy would be even worse in the global dashboard, where there are fewer tabs.

Reply:
For exp vs iso, this would get really complicated because this is in the css file now.  I think it's doable but not worth the complexity when it's not that much spacing we miss out on.

In terms of making the number dynamic based on the tab widths, that is so complex and totally not worth it

Reply:
That's fine with the dynamic approach, but are you also saying it's overly complex to have separate `max-width`s for each dashboard?

Reply:
I updated this here: https://github.com/SaraAlert/SaraAlert/pull/931/commits/c47b9730ab47045962e2e4a5e4a94bd3273a790e.

Should be good to go now besides potentially updating the "Records Requiring Review" language